sda-commons-server-kafka/src/test/java/org/sdase/commons/server/kafka/health/KafkaWithoutHealthCheckIT.java

```diff
@@ -26,24 +32,47 @@
           // fresh kafka instance
           .withBrokerProperty("group.initial.rebalance.delay.ms", "0");
 
-  private static final DropwizardAppRule<KafkaTestConfiguration> DW =
-      new DropwizardAppRule<>(
-          KafkaWithoutHealthCheckTestApplication.class,
-          resourceFilePath("test-config-default.yml"),
-          config("kafka.brokers", KAFKA::getKafkaConnectString));
+  private static final WireMockClassRule WIRE =
+      new WireMockClassRule(wireMockConfig().dynamicPort());
```

Comment:
Strange that wiremock is inside this test class? Or do I miss something?

Reply:
Was just WIP, turned out I didn't need it